src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
It shouldn't be a breaking change, as long as `shuffle` remains the same

```
val shuffle : xs:seq<'a> -> Gen<'a []>
```

and as long as it's properties also remain the same.

That comment is there in order to, hopefully, help any code reader understand easier what's going on, if he or she is not aware of that particular algorithm.

Could we do better?